cmd/controller.go

```diff
@@ -38,17 +37,13 @@ var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
 
-	metricsAddr              string
-	enableLeaderElection     bool
-	healthProbeAddr          string
-	containerImage           string
-	monitoringStorageClasses []string
```

Comment:
With the removal of `monitoringStorageClasses`, the followings are should be updated.

https://github.com/topolvm/pie/blob/db39a5b5ab5592f4719d10b8a8843abd93567a7c/README.md?plain=1#L20-L27

https://github.com/topolvm/pie/blob/db39a5b5ab5592f4719d10b8a8843abd93567a7c/config/samples/pie_v1alpha1_pieprobe.yaml#L14-L15